source/ControllableHero.hx

```diff
@@ -0,0 +1,48 @@
+package;
+
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+import flixel.FlxG;
+
+class ControllableHero extends FlxSprite
+{
+    public function new()
+        {
+            super();
+            makeGraphic(50, 45, FlxColor.WHITE, false);
+        }
+
+
+        override public function update(elapsed:Float) 
+        {
+            if (FlxG.keys.justPressed.UP)
+                {this.y += -1;}
+            if (FlxG.keys.pressed.UP)
+                {this.y += -1;}
+            if (FlxG.keys.justReleased.UP)
+                {this.y += -1;}
+        
+            if (FlxG.keys.justPressed.RIGHT)
+                {this.x += 1;}
```

Comment:
(This applies to all of these blocks)
This is a matter of coding style, but I don't recommend using this style of curly braces on either side of the body of the conditional. It does make the code more compact, but it comes with some drawbacks:
1. Curly braces (which are used for organizing the logical flow of our program) are right next to code that's defining what that logic is. From my perspective, placing them side by side puts the extra burden of separating the two (control flow and logic) on the folks reading it, instead of trying to keep the two separate where possible.
2. I imagine that if we wanted to add to any of these logical blocks, we'd have to do some work to manually untangle them. It's not a problem if this code never changes, but it's a cost that's worth considering, especially in an area of code that's as high-traffic as our `update()` method.

The furthest I recommend slimming down `if` statements is something like 
```
if (FlxG.keys.justPressed.RIGHT) {
    this.x += 1;
}
```
This keeps logic cleanly separated from control, hiding the first `{` on the `if` line since coding convention & actual syntactical rules dictate that opening curly braces will always immediately follow that `if`.

If you want to go further, I've seen folks take blocks of if / else if / else type blocks and organize them like:
```
if (FlxG.keys.justPressed.LEFT) {
    this.x += -1;
} else if (FlxG.keys.pressed.LEFT) {
    this.x += -1;
} else if (FlxG.keys.justReleased.LEFT) {
    this.x += -1;
}
```
I personally find this harder to read, but it does stick to the concept of separating code logic and control flow. I wouldn't suggest doing it if you have a bunch of different `if`s, though, I think the grouping implies that all of these conditionals are in one "family"

Since it is a matter of style, no change is required here. I strongly recommend making some kind of adjustment though.